error-handling/error-handling.js

```diff
@@ -1,3 +1,20 @@
+exports.handlePsqlErrors = (err, req, res, next) => {
+  const validErrors = ["22P02"];
+  if (validErrors.includes(err.code)) {
+    res.status(400).send({ msg: "bad request" });
```

Comment:
nicely done! you might just want to keep an eye on this though, in case you get a PSQL error further down the line which isn't a 400 👍